src/Components/Layout/Layout.tsx

```diff
@@ -21,9 +22,10 @@ export const Layout = ({
     <>
       <Helmet title={title} />
       <ChakraProvider theme={theme}>
+        <Fonts />
         <Flex minHeight='100vh' direction='column'>
           <Navbar />
-          <Flex flex='1' direction='column' px={5} align='center'>
+          <Flex flex='1' direction='column' p={16} align='center'>
```

Comment:
Changed this padding to match that of the navbar for now.

Also just a general note, [Chakra uses a spacing scale where 1 unit = 0.25 rem, which is 4 px in most browsers](https://chakra-ui.com/docs/theming/theme#spacing). So here, the padding is set to 64 px.